src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -104,14 +105,15 @@ class ResumeCtrlTest : public ::testing::Test {
   application_manager_test::MockApplicationManagerSettings
       mock_application_manager_settings_;
   application_manager_test::MockApplicationManager app_mngr_;
+  MockStateController state_controller_;
   utils::SharedPtr<ResumeCtrl> res_ctrl;
   utils::SharedPtr<NiceMock<resumption_test::MockResumptionData>> mock_storage;
   utils::SharedPtr<NiceMock<MockApplication>> app_mock;
   // app_mock.app_id() will return this value
   uint32_t test_app_id;
   std::string test_policy_app_id;
   std::string mac_address;
-  mobile_apis::HMILevel::eType default_test_type;
+  mobile_apis::HMILevel::eType default_test_level;
```

Comment:
Variable `mobile_apis::HMILevel::eType default_test_level` never get change. Make it `const` and rename according to coding guideline.